vendor/github.com/docker/libnetwork/drivers/overlay/overlayutils/utils.go

```diff
@@ -0,0 +1,44 @@
+// Package overlayutils provides utility functions for overlay networks
+package overlayutils
+
+import (
+	"fmt"
+	"sync"
+)
+
+var (
+	vxlanUDPPort uint32
+	mutex        sync.Mutex
+)
+
+func init() {
```

Comment:
Can you create a constant for 4789?

```suggestion
const defaultVXLANUDPPort = 4789
func init() {
 	vxlanUDPPort = defaultVXLANUDPPort
}
```